test/common/helper.mligo

```diff
@@ -30,7 +30,7 @@ let init_storage (owners, threshold: address set * nat) : storage_types =
   owners           = owners;
   threshold        = threshold;
   effective_period = 172800;
-  metadata         = (Big_map.empty: (string, bytes) big_map);
+  metadata         = Big_map.literal [("", 0x01)];
```

Comment:
Why is not empty?

Reply:
This is initial value of storage for running creating a contract. Usually, users will add something to metadata at this time. 
There was no way to modify it once creation is done so it was not really important thing to test.
Now, we have a way to update it. Also, as it's just a test, so I put random value there for testing.